Gemfile

```diff
@@ -1,4 +1,4 @@
-source :rubygems
+source 'https://rubygems.org'
 
 gem 'activesupport', '>= 2.3.11'
```

Comment:
should we bump the version or will bundler automatically pick up the new version?